src/Common/src/Interop/Windows/NtDll/Interop.NtCreateFile.cs

```diff
@@ -0,0 +1,580 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class NtDll
+    {
+        // https://msdn.microsoft.com/en-us/library/bb432380.aspx
+        // https://msdn.microsoft.com/en-us/library/windows/hardware/ff566424.aspx
+        [DllImport(Libraries.NtDll, CharSet = CharSet.Unicode, ExactSpelling = true)]
+        private unsafe static extern int NtCreateFile(
```

Comment:
why not define it to return Interop.StatusOptions (NTSTATUS)

BTW are we consistent when we use "original" name (seems structs and constants) vs "C# style" name (seems enum here)

Reply:
Because it isn't an enum. I'd prefer it to be. I can follow up with another change that does this- I'd have to change some other projects.